feat: symlinks evaluation in long mode

canta2899 opened the pull request:
Hi!

I implemented a way to evaluate symlinks in long mode, that behaves like the original ls. When the file is a symlink, logo-ls prints the **symlink path** followed by a `  -> ` symbol and the **path of the original file**.

Here's an example of the output:

<img width="1042" alt="image" src="https://user-images.githubusercontent.com/56071114/173356321-9791ac63-9ca7-438d-9f12-6c6ed2beb836.png">

I also had to correct a few **case sensitive file names** because I'm working on OS X (which has case insensitive paths) and I kept getting weird errors that did not allow me to stage changes.

---

**Note**:
I opened another PR a few months ago (#35 which is still open and not merged). In order to avoid issues I developed this feature on another branch so that both the PRs can be evaluated independently and eventually merged with ease.

canta2899 committed:
feat: Symlinks evaluation in long mode

canta2899 committed:
fix: Removing case sensitive paths

svenlowry-cgi commented:
Can this get approved and merged? Had to stop using logo-ls because I need to be able to see simlink destinations in ls commands!

canta2899 commented:
@svenlowrybjss If you want, I have a fork of logo-ls on my profile you can pull from until the PR has been (let's hope so) approved. So far I fixed a few issues and implemented this additional feature.
